src/scss/06_components/paragraphs/_contact-card.scss

```diff
@@ -0,0 +1,78 @@
+.contact-card {
+  display: flex;
+}
+
+.contact-card--with-image {
+  @include breakpoint($breakpoint-s) {
+    flex-direction: column;
+  }
+}
+
+.contact-card--information-container {
+  @include breakpoint($breakpoint-s) {
+    max-width: 200px;
+  }
+}
+
+.contact-card--person-social-media {
+  display: flex;
+
+  .hel-icon {
+    height: 26px;
+    margin-right: $spacing-half;
+    width: 26px;
+  }
+}
+
+.contact-card--image-container {
+  margin-bottom: $spacing;
+  margin-right: $spacing;
+
+  @include breakpoint($breakpoint-s) {
+    margin-right: 0;
+  }
+
+  img {
+    height: 100px;
+    width: 100px;
+
+    @include breakpoint($breakpoint-s) {
+      height: 200px;
+      width: 200px;
+    }
+  }
+}
+
+.contact-card--person-name {
+  font-size: 20px;
+  font-weight: $font-weight-medium;
+  margin-top: 0;
+}
+
+.contact-card--person-phone-number {
+  a:not(:last-of-type) {
+    &::after {
+      content: ',';
+      margin-right: 2px;
+    }
+  }
+}
+
+.contact-card--person-email {
+  word-break: break-all;
```

Comment:
I think `overflow-wrap: break-all` is a better alternative as it's not meant to break unless needed. Also, I think we have that for all links on by default: [overflow-wrap: break-word;](https://github.com/City-of-Helsinki/drupal-hdbt/blob/7a6d60e0be5fae32161d5711ed222d0616dd19c6/src/scss/04_elements/_typography.scss#L49-L50)

Reply:
That is true but in a case where there is this type of email "kyosti-pentti.hermanninpoika@hel.fi" the word break happens from the dash and not at the end of the line. In the design this type of email was broken like 

kyosti-pentti.hemmanninpoik
a@hel.fi 

So in order for this to happen word-break: break-all; is needed

Reply:
Oh, okay. I'm not sure if the designer has meant it to behave excatly like that, but both work I think.